### PR TITLE
*: add comment to make linter happy

### PR DESCRIPTION
**Pull request by lonng**
Signed-off-by: Lonng <chris@lonng.org>

### What problem does this PR solve? 

Linter will report some warning in no comment with exported elements.

### What is changed and how it works?

Add comment to make linter happy

### Check List 

Tests 

 - Unit test

Side effects

N/A

Related changes

N/A

### CODE REVIEW

File: runtime_test.go [hunk 1]
```diff
@@ -122,11 +128,75 @@ func (s *runtimeSuite) TestRuntime(c *C) {
...
 	c.Assert(ok, IsTrue)
 	c.Assert(val.(bool), IsTrue)
+
+	// Tests for sleep
+	ch = make(chan struct{})
+	go func() {
+		time.Sleep(time.Second)
+		err := failpoint.Disable("gofail/test-sleep")
+		c.Assert(err, IsNil)
+		close(ch)
```
Comment (kennytm) [inline runtime_test.go]: Change this to a `defer` at the beginning, so even if the assertion failed it the `<-ch` below won't block the process forever.

File: runtime_test.go [hunk 2]
```diff
@@ -122,11 +128,75 @@ func (s *runtimeSuite) TestRuntime(c *C) {
...
+	c.Assert(time.Since(start), GreaterEqual, 90*time.Millisecond, Commentf("not sleep"))
+	<-ch
+
+	// Tests for sleep duration
+	ch = make(chan struct{})
+	go func() {
+		time.Sleep(time.Second)
+		err := failpoint.Disable("gofail/test-sleep2")
+		c.Assert(err, IsNil)
+		close(ch)
```
Comment (kennytm) [inline runtime_test.go]: (ditto)

File: runtime_test.go [hunk 3]
```diff
@@ -122,11 +128,75 @@ func (s *runtimeSuite) TestRuntime(c *C) {
...
+	os.Stdout = w
+	err = failpoint.Enable("test-print", `print`)
+	ok, val = failpoint.Eval("test-print")
+	c.Assert(ok, IsFalse)
+	c.Assert(val, IsNil)
+	outC := make(chan string)
+	// copy the output in a separate goroutine so printing can't block indefinitely
+	go func() {
+		var buf bytes.Buffer
+		io.Copy(&buf, r)
```
Comment (kennytm) [inline runtime_test.go]: `io.Copy` returns an error, should need to handle this?

And perhaps you could just use `ioutils.ReadAll`

**Comment by lonng** [comment]
@amyangfei @kennytm PTAL again

**Pull request merged**